python/pyfunc-server/pyfuncserver/model/model.py

```diff
@@ -44,12 +48,13 @@ def _is_latest_pyfunc_model_version(func):
     return len(full_args.args) == NUM_OF_LATEST_PREDICT_FUNC_ARGS
 
 
-class PyFuncModel(kfserving.KFModel):  # pylint:disable=c-extension-no-member
+class PyFuncModel:  # pylint:disable=c-extension-no-member
```

Comment:
why not derived from `kfserving.KFModel` anymore?

Reply:
I am removing dependency from kfserving since we are not using many of its feature. Additionally it help us to reduce number of dependency that we need to maintain.